server.py

```diff
@@ -21,16 +21,31 @@ def generate(r):
     for chunk in r.iter_content(chunk_size):
         yield chunk
 
+def is_image(out_path):
+    return out_path.endswith('.jpg') or out_path.endswith('.jpeg') or out_path.endswith('.png') or out_path.endswith('.gif')
+
 def fetch_image(out_path):
     r = requests.get(out_path, stream=True, params=request.args)
     headers = dict(r.headers)
 
+    if headers['Content-Type']  == "binary/octet-stream" and is_image(out_path):
+            headers['Content-Type'] = "image/jpeg"
+    if app.debug:
+        print('serving with headers ' + str(headers))
```

Comment:
in general there should be some kind of logger where you can output stuff on debug level, but i don't know how it's done normally within these project

Reply:
i copied this from other parts in this file. i'd also prefer a logger but not sure if it's worth to od it